lighthouse-cli/test/smokehouse/smokehouse.js

```diff
@@ -133,15 +187,25 @@ function collateResults(actual, expected) {
 
 /**
  * Log the result of an assertion of actual and expected results.
- * @param {{category: string, equal: boolean, actual: boolean, expected: boolean}} assertion
+ * @param {{category: string, equal: boolean, diff: ?Object, actual: boolean, expected: boolean}} assertion
  */
 function reportAssertion(assertion) {
   if (assertion.equal) {
     console.log(`  ${log.greenify(log.tick)} ${assertion.category}: ` +
         log.greenify(assertion.actual));
   } else {
-    console.log(`  ${log.redify(log.cross)} ${assertion.category}: ` +
-        log.redify(`found ${assertion.actual}, expected ${assertion.expected}`));
+    if (!assertion.diff) {
```

Comment:
can you flip this if/else? had to pause for what this was meaning `!equal && !diff === 'diff was just missing'` :)